src/components/button/Button.base.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import Grid, { Spacing, TSpacingTokensSymmetric } from '../../foundations/layout';
+import Shape from '../../foundations/shape';
+import Icon, { TIconSize } from '../icon';
+import Text, { TTextSizeToken } from '../text';
+
+import { PButton } from './Button.props';
+
+const ButtonBase: React.FC<PButton> = ({
+    label,
+    leadingIcon,
+    trailingIcon,
```

Comment:
The Design System doesn't support simultaneous leading and trailing icons (I double checked). We should either logically exclude a trailing icon if a leading icon is also defined, or we should change this to use different props like an `icon` and `iconSide` prop instead.

eg. `iconGlyph: TIconGlyph` & `iconSide: 'left' | 'right'`

If we don't limit this, people will start to use dual icons on buttons against the Design System docs.

Reply:
I will keep it as is for now, since I have another idea i would like to discuss with you first.